org.eclipse.m2e.jdt/src/org/eclipse/m2e/jdt/internal/BuildPathManager.java

```diff
@@ -173,6 +173,21 @@ public static IClasspathEntry getMavenContainerEntry(IJavaProject javaProject) {
     return null;
   }
 
+  public static IClasspathEntry getJREContainerEntry(IJavaProject javaProject) {
```

Comment:
move to MavenClasspathHelpers

Reply:
ok. What about getMavenContainerEntry, isn't it a bit similar ?